extensions/objects/models/objects.py

```diff
@@ -1035,3 +1035,70 @@ class DragAndDropPositiveInt(BaseObject):
     default_value = 1
 
     SCHEMA = PositiveInt.SCHEMA
+
+
+class AlgebraicExpression(BaseObject):
+    """Class for algebraic expressions. Stores a unicode string representing a
+    valid algebraic expression.
+    """
+
+    description = 'A unicode string for an algebraic expression.'
+    default_value = 'a'
+
+    SCHEMA = {
+        'type': 'unicode',
+        'validators': [{
+            'id': 'is_valid_expression',
+            'algebraic': True
+        }]
+    }
+
+
+class SetOfPlaceholders(BaseObject):
+    """Class for set of placeholders. Placeholders are latin letters that are
```

Comment:
Would you include greek letters?

Reply:
Yes, the user will be able to input greek letters but since the format is ASCIIMath, we won't need to include special characters. For instance, if the user inputs `α`, the ASCIIMath format will be `alpha`. You can try out the conversions [here](https://guppy.js.org/site/), just make sure you've selected ASCIIMath.

Reply:
Your reply here doesn't seem to jive with "is_each_element_a_single_latin_letter".

Reply:
Umm, I was referring to the latin alphabet with only 26 letters a-z (case insensitive). Should I change it to `is_each_element_a_single_english_alphabet` instead?

Reply:
No, I meant, it wouldn't be a *single* letter, right?

Reply:
Ah, yes, I've modified it now.

Reply:
The description of what placeholders are still seems wrong.

Reply:
Updated the description.